src/styles/images/representative/index.njk

```diff
@@ -8,6 +8,6 @@ stylesheets:
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-l">Do you have a V11 reminder letter printed in your name?</h1>
-    <img src="v11.gif" alt="">
+    <img src="v11.gif" alt="Illustration of a V11 reminder letter, showing the light blue letterhead area at the top, a white circle in the top-right corner and a few paragraphs of text">
```

Comment:
This alt text is not great.

It shouldn't start with "Illustration of" (which is just another way of saying "image of", which is against the guidance).

I cannot be sure because the image is missing context, but I assume it is meant to be decorative. A description only makes sense if it is important to know how the letter looks. But in that case the image should be a much better representation of the letter.

Instead of changing the alt text, I wonder if there is a better example you could use?

Reply:
Hey @selfthinker! I've been thinking this over, and recalling the context we were imagining for this example.

When noting that it's an illustration, the thought was that there's some value in distinguishing it from a photograph of a reminder letter.

And I think for the context, the image was intended to be informative, as it would show distinguishing factors of the V11 reminder letter (especially the blue letterhead).

That being said, we mainly added alt text to this example, because it had none. Yet there are important distinguishing visual details in the image that someone who can't see the image can then use to describe the letter to others.

Reply:
Maybe we should add context around this image. I wouldn't say that missing alt text is necessarily a problem, it might be, it really depends on the context.